bin/lint.sh

```diff
@@ -1,23 +1,88 @@
-#!/bin/bash
+#!/bin/sh
 
-set -euo pipefail
+set -e
```

Comment:
Just because I took it from https://github.com/mozilla/jira-bugzilla-integration/blob/main/bin/lint.sh

Reply:
Well, initially I was going to suggest keeping this the way it is and migrating JBI to ["bash strict mode"](http://redsymbol.net/articles/unofficial-bash-strict-mode/).

But turns out there are [some gotchas](https://mywiki.wooledge.org/BashFAQ/105) with that approach as well. 

So, maybe `-euo pipefail` is "safer", if not too safe, but this could also be fine as is, since I'm less convinced about the safety that these flags provide.